src/main.rs

```diff
@@ -41,20 +41,40 @@ async fn main() -> anyhow::Result<()> {
             .map(|streamer_message| handle_streamer_message(streamer_message, &pool))
             .buffer_unordered(1usize);
 
-        while let Some(_handle_message) = handlers.next().await {}
+        let mut time_now = std::time::Instant::now();
+        while let Some(handle_message) = handlers.next().await {
+            match handle_message {
+                Ok(block_height) => {
+                    let elapsed = time_now.elapsed();
+                    tracing::info!(
+                        target: LOGGING_PREFIX,
+                        "Elapsed time spent on block {}: {:.3?}",
```

Comment:
True
But I decided to leave it as it is because at that moment we haven't finished with metrics
When we'll be sure the metrics work fine, we can drop it